src/sql/src/util.rs

```diff
@@ -47,29 +48,16 @@ pub fn format_raw_object_name(name: &ObjectName) -> String {
     format!("{}", Inner { name })
 }
 
-pub fn parse_option_string(value: Expr) -> Option<String> {
-    match value {
-        Expr::Value(Value::SingleQuotedString(v)) | Expr::Value(Value::DoubleQuotedString(v)) => {
-            Some(v)
-        }
-        _ => None,
-    }
-}
-
-/// Converts options to HashMap<String, String>.
-/// All keys are lowercase.
-pub fn to_lowercase_options_map(opts: &[SqlOption]) -> HashMap<String, String> {
-    let mut map = HashMap::with_capacity(opts.len());
-    for SqlOption { name, value } in opts {
-        let value_str = match value {
-            Expr::Value(Value::SingleQuotedString(s))
-            | Expr::Value(Value::DoubleQuotedString(s)) => s.clone(),
-            Expr::Identifier(i) => i.value.clone(),
-            _ => value.to_string(),
-        };
-        let _ = map.insert(name.value.to_lowercase().clone(), value_str);
-    }
-    map
+pub fn parse_option_string(option: SqlOption) -> Result<(String, String)> {
+    let (key, value) = (option.name, option.value);
+    let v = match value {
+        Expr::Value(Value::SingleQuotedString(v)) | Expr::Value(Value::DoubleQuotedString(v)) => v,
+        Expr::Identifier(v) => v.value,
+        Expr::Value(Value::Number(v, l)) => format!("{}{long}", v, long = if l { "L" } else { "" }),
```

Comment:
This keeps the previous manner:

```rust
pub fn to_lowercase_options_map(opts: &[SqlOption]) -> HashMap<String, String> {
    let mut map = HashMap::with_capacity(opts.len());
    for SqlOption { name, value } in opts {
        let value_str = match value {
            Expr::Value(Value::SingleQuotedString(s))
            | Expr::Value(Value::DoubleQuotedString(s)) => s.clone(),
            Expr::Identifier(i) => i.value.clone(),
            _ => value.to_string(),
        };
        let _ = map.insert(name.value.to_lowercase().clone(), value_str);
    }
    map
}
```

`value.to_string()` will act as this line underneath.

But you're right that we may always drop the suffix since it won't help how we parse numbers.

Reply:
```suggestion
        Expr::Value(Value::Number(v, _)) => format!("{v}"),
```